cms/bundles/panels.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING, Any, Union
+
+from django.utils.html import format_html, format_html_join
+from django.utils.safestring import mark_safe
+from django.utils.translation import gettext as _
+from wagtail.admin.panels import HelpPanel, PageChooserPanel
+
+if TYPE_CHECKING:
+    from django.db.models import Model
+    from django.utils.safestring import SafeString
+
+
+class BundleNotePanel(HelpPanel):
+    """An extended HelpPanel class."""
+
+    class BoundPanel(HelpPanel.BoundPanel):
+        def __init__(self, **kwargs: Any) -> None:
+            super().__init__(**kwargs)
+            self.content = self._content_for_instance(self.instance)
+
+        def _content_for_instance(self, instance: "Model") -> Union[str, "SafeString"]:
+            if not hasattr(instance, "bundles"):
+                return ""
+
+            if bundles := instance.bundles:
+                content_html = format_html_join(
+                    "\n",
+                    "<li>{} (Status: {})</li>",
+                    (
+                        (
+                            bundle.name,
+                            bundle.get_status_display(),
+                        )
+                        for bundle in bundles
+                    ),
+                )
+
+                content = format_html(
+                    "<p>{}</p><ul>{}</ul>", _("This page is in the following bundle(s):"), content_html
+                )
+            else:
+                content = mark_safe(f"<p>{_('This page is not part of any bundles.')}</p>")  # noqa: S308
```

Comment:
Issue: Now that you're formatting the translation in, it _should_ be `format_html` :stuck_out_tongue:

Reply:
it _should_, but _doesn't have to_ :D
465b1c4